### ISSUE TITLE
Alt text and Accessibility for Images

### ISSUE THREAD

**Issue by benstorlie** [benstorlie/horiseon#3]
Currently, there is no alt text or consideration for screen readers on any of the given images.  I want to learn about accessibility guidelines and decide the appropriate changes to make, given the purposes of the images.

**Comment by benstorlie** [comment]
This article, ["What is pure decoration alt text in WCAG"](https://www.davidmacd.com/blog/what-is-pure-decoration-alt-text-in-wcag.htmll), has a lot of discussion on decorative images, and appropriate use of null alt text, and their gray zones and controversies.

Since there is a gray zone, it can be appropriate to add short image descriptions for the given stock photos, to add some "color" to the webpage (so to speak), and as a courtesy so the user knows what's there.  

On the other hand, this article, ["Alternate text for background images"](http://www.davidmacd.com/blog/alternate-text-for-css-background-images.html), discusses background images in CSS.  I haven't fully examined the whole article yet, but here is the takeaway:

> * Try not to use CSS for important informational images
> * For ambient images that are CSS, it is a courtesy to provide alternate text. When doing so, place image in its own empty `<span>` with an `aria-label` and `role="img"`. This is also true, in a situation where CSS must be used for information content.

Since the background image is already in its own `<div>`, the following might work just fine.

    <div class="hero" role="img" aria-label="[alt text here]"></div>
